docs/adr/0006-monitoring-structure.md

```diff
@@ -0,0 +1,69 @@
+# Application Monitoring in Operate First using Prometheus
+
+* Status: accepted
+* Deciders: 4n4nd, anishasthana, hemajv, HumairAK, tumido
+* Date: 2021-01-14
+
+Technical Story: [issue-1](https://github.com/operate-first/blueprint/issues/8)
+                 [issue-2](https://github.com/operate-first/apps/issues/57)
+
+## Context and Problem Statement
+
+Since there will be multiple services deployed in the Operate First environment
+(ex. Jupyterhub, Argo, Superset, Observatorium, etc) distributed over various
+namespaces, we need to be able to monitor them.
+
+To do so we will use [Prometheus](https://prometheus.io/).
+This will be deployed using the [ODH Operator](https://opendatahub.io/) (ODH
+operator includes the Prometheus Operator to do this).
+
+Normally, Prometheus Operator would have been able to monitor services across
```

Comment:
I feel like we should also mention User Workload Monitoring

Reply:
I think since cluster-monitoring and UWM both use the same instance of Proemtheus-Operator, it is okay to to not include it here as the conflict is with a second instance of the operator and not UWM.

Reply:
I'm commenting more from a "as of now UWM is not a common thing, but it will be in the near future"

Reply:
added something for it